src/steps/10-FinancialDetails/IGCE/SurgeCapabilities.vue

```diff
@@ -3,34 +3,36 @@
     <v-row>
       <v-col class="col-12">
         <h1 class="page-header">
-          Tell us more about the scope of your project
+          Tell us more about your potential surge capacity
         </h1>
 
         <ATATAlert
           type="callout"
           :showIcon="false"
-          class="copy-max-width mt-10"
+          class="copy-max-width pa-6"
+          calloutBackground="primary-lighter" 
         >
           <template v-slot:content>
-            <h2>Surge Capabilities</h2>
-            <p class="mt-2">
-              The Government may require surge capabilities during the base or
-              any option period, and surge modifications will be within the
-              scope of the contract for the defined task areas in the
-              description of work.
-            </p>
-            <p class="mb-0">
-              Surge capabilities over the life of the task order cannot exceed
-              between 1-50% of the contractor’s total proposed price for the
-              base and all option periods, excluding any six-month extension of
-              services pursuant to Federal Acquisition Regulation (FAR)
-              52.217-8.
+            <h2 class="mb-2">Surge Capabilities</h2>
+            <p class="ma-0">
+             Surge capabilities over the life of the task order can be between 
+             1-50% of the contractor’s total proposed price for the base and 
+             all option periods, excluding any six-month extension of services 
+             pursuant to 
+                <a id="FARlink"
+                role="button"
+                target="_blank"
+                tabindex="0"
+                href="https://www.acquisition.gov/far/52.217-8">
+              <span class="">Federal Acquisition Regulation (FAR) 52.217-8</span>
```

Comment:
add class `_external-link` to this span to get the "external link` icon after - it's not in the Figma but should be